android/src/main/java/com/heapanalytics/reactnative/RNHeap.java

```diff
@@ -0,0 +1,12 @@
+package com.heapanalytics.reactnative;
+
+import android.content.Context;
+import com.heapanalytics.android.Heap;
+import com.heapanalytics.android.internal.HeapImpl;
+
+public class RNHeap {
+    public static void init(Context context, String appId) {
```

Comment:
This is interesting!  It's a new file, right?  Do all classes that expose an `init` method with this sig get called on app initialization or something?  Is this magic, or are we calling it explicitly?  Just super curious!  (I know, I know, I "wrote" the stuff to begin with, but believe me FAR from an Android maven)

Reply:
It's a new file, yes, because I didn't want to put code that wasn't called from JS-land in the existing `RNHeapLibraryModule` class. Also this is not magic, we call this function from the context provider (HeapInitializationProvider.java).